infrastructure/scripts/trigger/start.sh

```diff
@@ -24,15 +24,10 @@ wait-for-service startup-helper schema-registry 8081 15 "Schema registry"
 
 echo "Starting up Airy Core Platform appplications"
 kubectl scale deployment -l type=api --replicas=1
-kubectl scale deployment -l type=sources-chatplugin --replicas=1
-kubectl scale deployment -l type=frontend --replicas=1
 
 wait-for-service startup-helper api-auth 80 10 api-auth
 
-kubectl scale deployment -l type=sources-twilio --replicas=1
-kubectl scale deployment -l type=sources-google --replicas=1
-kubectl scale deployment -l type=sources-facebook --replicas=1
-kubectl scale deployment -l type=webhook --replicas=1
+kubectl scale deployment -l app=airy-controller --replicas=1
```

Comment:
We scale the controller at the end, once everything is running and that is when all the optional apps would be started (even though the configMaps are deployed before)